examples/client/tui/commands.nim

```diff
@@ -780,11 +778,144 @@ proc command*(self: Tui, command: CallRpc) {.async, gcsafe,
     if command.rpcMethod == "":
       self.wprintFormatError(getTime().toUnix,
         "method cannot be blank, please provide an input.")
-    elif command.params.kind == JNull:
+      return
+    
+    var params = newJArray()
+    if command.params != "":
+      try:
+        params = command.params.parseJson()
+      except:
+        self.wprintFormatError(getTime().toUnix,
+          "params must be a valid JSON")
+        return
+
+    asyncSpawn self.client.callRpc(command.rpcMethod, params)
+  except:
+    self.wprintFormatError(getTime().toUnix, "invalid arguments.")
+
+# SendTransaction --------------------------------------------------------------
+
+proc help*(T: type SendTransaction): HelpText =
+  let command = "sendtransaction"
+  HelpText(command: command, aliases: aliased[command], parameters: @[
```

Comment:
```suggestion
  HelpText(command: command, aliases: aliased.getOrDefault(command), parameters: @[
```

My apologies for another requested change; this one is re: bugfix #277, which just landed.